ch5-catdir/utils.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+#include <sys/types.h>
+
+/*
+ * fmtMode - a function that converts a mode_t value to its graphical
+ *           representation (eg. 'drwxr-xr-x')
```

Comment:
"A functions that" is superfluous, in my eyes.
Also, e.g. (currently a dot is missing)

Reply:
It's fixed.